src/ggml-metal.h

```diff
@@ -109,6 +109,9 @@ GGML_API ggml_backend_buffer_type_t ggml_backend_metal_buffer_type(void);
 // ref: https://developer.apple.com/metal/Metal-Feature-Set-Tables.pdf
 GGML_API bool ggml_backend_metal_supports_family(ggml_backend_t backend, int family);
 
+// capture all command buffers committed the next time `ggml_backend_graph_compute` is called
+GGML_API void ggml_backend_metal_capture_next_compute(ggml_backend_t backend);
```

Comment:
@slaren Is it a good idea to extend the backend API with these type of calls or should we try to avoid it. In this case, it could remain a compile-time option (https://github.com/ggerganov/llama.cpp/commit/2a7c94db5fb67b2f8882d2d16a11bf5d8d12d397), but it might be useful to control at runtime in some cases

Reply:
Generally I don't see an issue with adding more backend-specific functions, as long as they are not required for normal usage.